spec/models/event_user_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EventUser do
+  let!(:user) { create(:user) }
+  let!(:event) { create(:event) }
+
+  context 'when belong to user and event' do
+    it { is_expected.to belong_to(:user) }
+    it { is_expected.to belong_to(:event) }
+  end
+
+  context 'when has a valid factory' do
+    it {
+      expect(create(:event_user,
+                    user_id: user.id,
```

Comment:
After you complete [this](https://github.com/Dunaysky/event-manager/pull/2/files#r1105161756) comment. You can create even_user without passing event and user variables

Reply:
Simplyfied creation of valid `event_user`: 
`it { expect(create(:event_user)).to be_valid }`